### ISSUE TITLE
Fix keyboard focus for tooltip group headers

### ISSUE THREAD

**Issue by smbea** [bpmn-io/properties-panel#269]
### Describe the Bug

I order to fix https://github.com/bpmn-io/properties-panel/issues/266, we move the tooltip to outside the group header containing `position: sticky`. This made it so the tooltip content isn't reachable in the keyboard focus tree: links can't be focused, for example.

### Steps to Reproduce

1. use keyboard to focus group label
2. 'tab' again and see that it skips the tooltip selectable content

### Expected Behavior

Tooltip content should be reachable via focus tree.

### Environment

- Host (Browser/Node version), if applicable: [e.g. MS Edge 18, Chrome 69, Node 10 LTS]
- OS: [e.g. Windows 7]
- Library version: [e.g. 2.0.0]
